src/modules/dashboard/components/building-list.vue

```diff
@@ -178,6 +195,23 @@ export default Vue.extend<
                 noSort?: boolean;
             }
         >;
+        const headingsHospital = (
+            this.listType === 'building' &&
+            useTranslationStore().bedBuildings.includes(
+                this.buildings[0]?.building_type
+            )
+                ? {
+                      beds: { title: this.$m('beds'), noSort: false },
+                      bedsFree: { title: this.$m('bedsFree'), noSort: false },
```

Comment:
sorting is not respected in `buildingsSorted`.

Reply:
So I can remove the parameter, because it's ignored anyway?